src/sage/schemes/elliptic_curves/saturation.py

```diff
@@ -383,9 +383,54 @@ def p_saturation(self, Plist, p, sieve=True):
             sage: saturator = EllipticCurveSaturator(E, True)
             sage: saturator.p_saturation([P, T], 2)
             Using sieve method to saturate...
-            ...
+            E has 2-torsion over Finite Field of size 13, projecting points
+             --> [(0 : 2 : 1), (0 : 1 : 0)]
+             --rank is now 0
+            E has 2-torsion over Finite Field of size 13, projecting points
+             --> [(10 : 6 : 1), (0 : 1 : 0)]
+             --rank is now 1
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 23, projecting points
+             --> [(6 : 14 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 23, projecting points
+             --> [(5 : 17 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 41, projecting points
+             --> [(17 : 25 : 1), (0 : 1 : 0)]
+             --rank is now 1
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 41, projecting points
+             --> [(33 : 21 : 1), (0 : 1 : 0)]
+             --rank is now 1
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 53, projecting points
+             --> [(31 : 4 : 1), (0 : 1 : 0)]
+             --rank is now 1
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 53, projecting points
+             --> [(52 : 3 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 59, projecting points
+             --> [(48 : 39 : 1), (0 : 1 : 0)]
+             --rank is now 1
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 59, projecting points
+             --> [(31 : 33 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 73, projecting points
+             --> [(6 : 42 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            E has 2-torsion over Finite Field of size 73, projecting points
+             --> [(15 : 62 : 1), (0 : 1 : 0)]
+             --rank is now 1
+            ! rank same for 15 steps, checking kernel...
+            kernel vectors: [
+            (0, 1)
+            ]
+            points generating kernel: [(0 : 1 : 0)]
```

Comment:
Are we sure that this output is always the same? I'd assume there's a reason it was `...` previously?

Reply:
Sorry, I think it's not. I used `sage --fixdoctests` to perform this automatically, sorry for that!